coderd/coderdtest/coderdtest.go

```diff
@@ -614,6 +614,21 @@ func CreateAnotherUserMutators(t testing.TB, client *codersdk.Client, organizati
 	return createAnotherUserRetry(t, client, organizationID, 5, roles, mutators...)
 }
 
+// AuthzUserSubject does not include the user's groups.
+func AuthzUserSubject(user codersdk.User) rbac.Subject {
```

Comment:
That's handy!

Reply:
Yup, got tired of using `AsSystemContext` in all my unit tests. Figured this would be a bit better.